config/install/user.role.school_admin.yml

```diff
@@ -23,6 +23,7 @@ is_admin: null
 permissions:
   - 'access student_admission_workflow workflow_transition form'
   - 'access toolbar'
+  - 'administer entity print'
```

Comment:
Is this required for school admin?

Reply:
No, this will work without this. This was added for testing some points.I'll remove this.